components/workflow.rst

```diff
@@ -76,16 +76,19 @@ are trying to use it with::
 Usage
 -----
 
-When you have configured a ``Registry`` with your workflows, you may use it as follows::
+When you have configured a ``Registry`` with your workflows,
+you can retreive a workflow from it and use it as follows::
 
     // ...
-    $post = new BlogPost();
     $workflow = $registry->get($post);
```

Comment:
now `$post` is undefined here 🤔 
Why do you move it below?

Reply:
a fail, wanted to move the new php comment